Packs/Linkshadow/Integrations/Linkshadow/Linkshadow.py

```diff
@@ -0,0 +1,209 @@
+    return anomaly_info
+
+
+def fetch_incidents(client, max_alerts, last_run, first_fetch_time, apiKey, api_username, plugin_id, action, time_frame):
+
+    last_fetch = last_run.get('last_fetch', None)
+    if last_fetch is None:
+        last_fetch = first_fetch_time
+    else:
+        last_fetch = int(last_fetch)
+    latest_created_time = int(last_fetch)
+    incidents = []
+    alerts = client.fetch_anomaly(apiKey=apiKey, api_username=api_username,
+                                  plugin_id=plugin_id, action=action, time_frame=time_frame)
+    for dic in alerts['data']:
+        for key in dic.keys():
+            if key == 'time_seen':
+                incident_created_time = dic['time_seen']
+                if last_fetch:
+                    if int(incident_created_time) <= last_fetch:
+                        continue
+                incident_name = "Linkshadow-entityAnomaly"
+                formatted_JSON = format_JSON_for_fetch_incidents(dic)
+
+                incident = {
+                    'name': incident_name,
+                    'occurred': timestamp_to_datestring(incident_created_time),
+                    'rawJSON': formatted_JSON,
+                    'CustomFields': {  # Map specific XSOAR Custom Fields
+                        'sip': formatted_JSON['sip'],
+                        'sourceip': formatted_JSON['sip'],
+                        'destinationip': formatted_JSON['dip'],
+                        'sourceport': formatted_JSON['sport'],
+                        'destinationport': formatted_JSON['dport'],
+                        'macaddress': formatted_JSON['smac'],
+                        'alertid': formatted_JSON['anomaly_id'],
+                        'subcategory': formatted_JSON['category']
+                    }
+                }
+                incidents.append(incident)
+                # Update last run and add incident if the incident is newer than last fetch
+                # if incident_created_time > latest_created_time:
+                #     latest_created_time = incident_created_time
+                # print (max_alerts)
+                if len(incidents) >= max_alerts:
+                    break
+    next_run = {'last_fetch': latest_created_time}
+    return next_run, incidents
+
+
+def fetch_entity_anomalies(client, args, arg):
+    apiKey = args.get('apiKey'),
+    username = args.get('api_username')
+    plugin_id = args.get('plugin_id')
+    action = args.get('action')
+
+    time_frame = arg_to_number(
+        arg=arg.get('time_frame'),
+        arg_name='time_frame',
+        required=False
+    )
+
+    alerts = client.fetch_anomaly(
+        apiKey=apiKey,
+        api_username=username,
+        plugin_id=plugin_id,
+        action=action,
+        time_frame=time_frame
+    )
+
+    alert = []
+    for dic in alerts.get('data'):
+        alert.append(dic)
+
+    if len(alert) == 0:
+        alert = [{"message": "Linkshadow Anomaly already acknowledged!!"}]

```

Comment:
Unnecessary. the outcome of the alert list will be the same as `alerts.get('data')`
```suggestion
    
```